WireAuthentication/Sources/WireAuthenticationUI/Views/Registration/AccountTypeSelection/AccountTypeSelectionViewModel.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+@MainActor
+package final class AccountTypeSelectionViewModel: ObservableObject {
+
+    @Published var isCreateTeamAccountPresented = false
+
+    private let teamsURL: URL
```

Comment:
suggestion: instead of storing this base URL privately and having a computed property based on it, why not create the resulting value on init and store that value?